addons/event/static/src/views/event_registration_kanban_controller.js

```diff
@@ -0,0 +1,60 @@
+import { kanbanView } from "@web/views/kanban/kanban_view";
+import { KanbanController } from "@web/views/kanban/kanban_controller";
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { onMounted, onWillUnmount } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+export default class EventRegistrationKanbanController extends KanbanController {
+
+    setup() {
+        super.setup()
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+        onMounted(this.hideSystray);
+        onWillUnmount(this.showSystray);
+    }
+
+    async openRecord(record, mode) {
+        if (this.props.context.is_from_barcode) {
+            const barcode = record.data.barcode
+            const eventId = record.data.event_id[0]
+
+            const result = await this.orm.call("event.registration", "register_attendee", [], {
+                barcode: barcode,
+                event_id: eventId,
+            });
+
+            this.dialog.add(
+                EventRegistrationSummaryDialog,
+                {
+                    registration: result,
+                    model: this.model
+                }
+            )
+        } else {
+            return super.openRecord(record, mode);
+        }
+    }
+
+    hideSystray() {
+        if (this.props.context.is_from_barcode) {
+            this.observer = new MutationObserver((mutations) => {
+                document.querySelector('.o_menu_systray').classList.add('d-none');
+                this.observer.disconnect();
+            });
+            this.observer.observe(document.body, { childList: true, subtree: true });
+        }
+    }
```

Comment:
I understand the idea but this does not check anything on the mutation and would crash if o_menu_systray is not present. I would still avoid it. Coming back to you if I find a better idea

Reply:
Maybe adding target fullscreen to both actions could work? (in the xml) Maybe to check with LNA (the whole top bar disappears) We also need to make sure we can exit those views if we do so. Otherwise we may drop this small point of the spec to move forward?